pkg/chunkenc/memchunk.go

```diff
@@ -35,6 +37,7 @@ var (
 // with any other use of the crc32 package anywhere. Thus we initialize it
 // before.
 var castagnoliTable *crc32.Table
+var logger log.Logger
```

Comment:
you need to use util.Logger instead which is initialized. This one is not :)

Reply:
Sure